packages/optimizer/lib/transformers/ApplyLayout.js

```diff
@@ -118,7 +118,7 @@ function maybeAddSizerInto(node, layout, width, height) {
 function createResponsiveSizer(width, height) {
   const padding = (height.numeral / width.numeral) * 100;
   const sizer = createElement('i-amphtml-sizer', {
-    style: `display:block;padding-top:${padding.toFixed(4)}%;`,
+    style: `display:block;padding-top:${padding.toFixed(4).replace(/\.?0*$/, '')}%`,
```

Comment:
any reason we aren't using `parseFloat`?

Reply:
Well, we already have a float, the problem is to format a float correctly into a string, not the other way around.

Reply:
well anything used here has toString implicitly being called on it  as a result of being referenced inside of a template literal.

Taking a float, `toFixed`-ing it, and then parsing it again would accomplish the same thing without a regex, right?

Reply:
That seems to be slower than using a regex, though:

![Image 2020-10-07 at 8 14 16 AM](https://user-images.githubusercontent.com/83631/95294292-4e225200-0875-11eb-8e5a-0a53a4b73a5a.png)

See https://www.measurethat.net/Benchmarks/Show/9870/0/float-string-optimization-parsefloat-vs-regex

Reply:
for sure! and great research, but it was more about the legibility of the code rather than the performance of several million parses. I am hard-pressed to think of a situation where that would be plausible

Reply:
Got it, I'll adapt the code.

Reply:
Resolved via https://github.com/ampproject/amp-toolbox/pull/930/commits/6ad6c9eac78caf49d57e23964049ca834ee92f0e